cirrus/server/cirrus/experiment_recipes.py

```diff
@@ -1,13 +1,32 @@
-from typing import Any, Dict, List
+import logging
+from typing import Any, Dict, List, cast
+
+import requests
+from decouple import config  # type: ignore
+
+logger = logging.getLogger(__name__)
 
 
 class RemoteSettings:
     recipes: List[Dict[str, Any]] = []
+    url: str = cast(str, config("REMOTE_SETTING_URL", default=""))
+
+    def get_recipes(self) -> List[Dict[str, Any]]:
+        return self.recipes
 
-    @classmethod
-    def get_recipes(cls) -> List[Dict[str, Any]]:
-        return cls.recipes
+    def update_recipes(self, new_recipes: List[Dict[str, Any]]) -> None:
+        self.recipes = new_recipes
 
-    @classmethod
-    def update_recipes(cls, new_recipes: List[Dict[str, Any]]) -> None:
-        cls.recipes = new_recipes
+    def fetch_recipes(self):
+        try:
+            response = requests.get(self.url)
+            if response.status_code == 200:
```

Comment:
Do you have to check this?

Reply:
just making sure status is 200, but can also do something like this 
```python
try:
    response = requests.get(url)
    response.raise_for_status()  # raises an HTTPError for bad status codes
    data = response.json().get("data", [])
    if data:
        # process the data
    else:
        print("No recipes found in the response")
except requests.exceptions.RequestException as e:
    print(f"Failed to fetch recipes: {e}")